nautobot/docs/user-guide/core-data-model/users/savedview.md

```diff
@@ -0,0 +1,46 @@
+# Saved Views
+
++++ 2.3.0
+
+Saved Views give users the ability to save multiple configurations of list views, including table columns, filtering, pagination, and sorting, for ease of later use and reuse. Check out this [user guide](../../feature-guides/saved-views.md) on how to use saved views.
```

Comment:
This is another case where I question why we have separate "model" and "user guide" documentation files. Why not combine them?